src/java/org/apache/cassandra/index/sai/disk/format/IndexComponent.java

```diff
@@ -79,10 +84,20 @@ public enum IndexComponent
      */
     GROUP_COMPLETION_MARKER("GroupComplete");
 
+    public final Component.Type type;
```

Comment:
super OCD nit: maybe `type` could go below `name` so they're in the same order in the ctor :D

Reply:
+1